unittest/resultiterator_test.cc

```diff
@@ -274,6 +274,10 @@ class ResultIteratorTest : public testing::Test {
 
 // Tests that Tesseract gets exactly the right answer on phototest.
 TEST_F(ResultIteratorTest, EasyTest) {
+#ifdef DISABLED_LEGACY_ENGINE
```

Comment:
This test is a layout analysis test which does not really need osd.

It should work with lstm, if the oem and psm are adapted.